modules/ROOT/pages/performance/memory-configuration.adoc

```diff
@@ -102,7 +102,7 @@ Estimate the total size of the database files.
 
 [source, shell]
 ----
-$neo4j-home> bin/neo4j-admin server memory-recommendation
+$NEO4J_HOME> bin/neo4j-admin server memory-recommendation
```

Comment:
just to double check: Do we need `$NEO4J_HOME>` here?

Reply:
No, of course not. Good catches! Replace without checking afterwards doesn't seem to work 😄